stdlib/LinearAlgebra/test/cholesky.jl

```diff
@@ -495,5 +495,70 @@ end
     @test B.L ≈ B32.L
     @test B.UL ≈ B32.UL
 end
+                                
+@testset "det and logdet" begin
+    A = [4083 3825 5876 2048 4470 5490;
+         3825 3575 5520 1920 4200 5140;
+         5876 5520 8427 2940 6410 7903;
+         2048 1920 2940 1008 2240 2740;
+         4470 4200 6410 2240 4875 6015;
+         5490 5140 7903 2740 6015 7370]
+    A1 = [4 12 -16;
+    12 37 -43;
+    -16 -43 98]
+    A2 = [ 6 15 55
+    15 55 225
+    55 225 979]
+    B = cholesky(A, Val(true), check=false)
+    B1 = cholesky(A1, Val(true))
+    B2 = cholesky(A2, Val(true))
+    @test det(B)  ≈  0.0
```

Comment:
This test seems new, but I assume this is a case with `C.rank < size(C.factors, 1)? Then this should be exactly zero:
```suggestion
    @test det(B) == 0.0
```
...